README.md

```diff
@@ -1,64 +1,61 @@
 > **Warning**
 > This project is in early development, it does however work with real sats! Always use amounts you don't mind losing.
 
-
 # Cashu Development Kit
 
 CDK is a collection of rust crates for [Cashu](https://github.com/cashubtc) wallets and mints written in Rust.
 
 **ALPHA** This library is in early development, the api will change and should be used with caution.
 
-
 ## Project structure
 
 The project is split up into several crates in the `crates/` directory:
 
-* Libraries:
-    * [**cdk**](./crates/cdk/): Rust implementation of Cashu protocol.
-    * [**cdk-sqlite**](./crates/cdk-sqlite/): SQLite Storage backend.
-    * [**cdk-redb**](./crates/cdk-redb/): Redb Storage backend.
-    * [**cdk-rexie**](./crates/cdk-rexie/): Rexie Storage backend for browsers.
-    * [**cdk-axum**](./crates/cdk-axum/): Axum webserver for mint.
-    * [**cdk-cln**](./crates/cdk-cln/): CLN Lightning backend for mint.
-    * [**cdk-lnd**](./crates/cdk-lnd/): Lnd Lightning backend for mint.
-    * [**cdk-strike**](./crates/cdk-strike/): Strike Lightning backend for mint.
-    * [**cdk-lnbits**](./crates/cdk-lnbits/): [LNbits](https://lnbits.com/) Lightning backend for mint.
-    * [**cdk-fake-wallet**](./crates/cdk-fake-wallet/): Fake Lightning backend for mint. To be used only for testing, quotes are automatically filled.
-* Binaries:
-    * [**cdk-cli**](./crates/cdk-cli/): Cashu wallet CLI.
-    * [**cdk-mintd**](./crates/cdk-mintd/): Cashu Mint Binary.
-
+- Libraries:
+  - [**cdk**](./crates/cdk/): Rust implementation of Cashu protocol.
+  - [**cdk-sqlite**](./crates/cdk-sqlite/): SQLite Storage backend.
+  - [**cdk-redb**](./crates/cdk-redb/): Redb Storage backend.
+  - [**cdk-rexie**](./crates/cdk-rexie/): Rexie Storage backend for browsers.
+  - [**cdk-axum**](./crates/cdk-axum/): Axum webserver for mint.
+  - [**cdk-cln**](./crates/cdk-cln/): CLN Lightning backend for mint.
+  - [**cdk-lnd**](./crates/cdk-lnd/): Lnd Lightning backend for mint.
+  - [**cdk-strike**](./crates/cdk-strike/): Strike Lightning backend for mint.
+  - [**cdk-lnbits**](./crates/cdk-lnbits/): [LNbits](https://lnbits.com/) Lightning backend for mint.
+  - [**cdk-phoenixd**](./crates/cdk-phoenixd/): Phoenixd Lightning backend for mint.
+  - [**cdk-fake-wallet**](./crates/cdk-fake-wallet/): Fake Lightning backend for mint. To be used only for testing, quotes are automatically filled.
+- Binaries:
+  - [**cdk-cli**](./crates/cdk-cli/): Cashu wallet CLI.
+  - [**cdk-mintd**](./crates/cdk-mintd/): Cashu Mint Binary.
```

Comment:
Is there a problem with using the `*`?

Reply:
> Is there a problem with using the `*`?

Thanks for catching. I was too confident with the change and didn't catch that my IDE reformatted the markdown

Reply:
Was thinking it was an IDE thing. If thats the standard in markdown I don't mind changing it. It renders the same not that its the prettiest things anyway. Thanks for the PR